### ISSUE TITLE
Error in function <SNR>43_fix_paths when compiling and opening quickfix

### ISSUE THREAD

**Issue by alefa** [lervag/vimtex#805]
### Explain the issue

After updating Vimtex to the newest version, I get error messages when I try to compile my document with \<localleader\>ll (single-shot compilation) and when I try to open the quickfix window with \<localleader\>le. Because I think the two errors might be related, I report them as one issue. I'm using Neovim 0.1.7 on Debian Stretch.

1. Steps to reproduce:
Open the minimal example (see below) in Neovim using the minimal init.vim (also below).
a) Press \<localleader\>ll. 
b) Press \<localleader\>le.

2. Expected behaviour:
a) The document should compile without error.
b) The quickfix window should open and show compiler errors and warnings.

3. Observed behaviour
a) An [error message](http://imgur.com/a/jZMEN) is shown, but the document is still compiled correctly.
b) An [error message](http://imgur.com/a/eZAWn) is shown. This happens also with a minimal init.vim without setting any Vimtex options. The quickfix window does not open. Opening it with the usual command (:copen) works correctly, however.

### Minimal working example

```tex
\documentclass{article}

\title{A test document}

\begin{document}

\maketitle	

Hello, world!

\end{document}
```

### Minimal init.vim file

```vim
syntax on

call plug#begin('~/.config/nvim/plugged')
Plug 'lervag/vimtex'
call plug#end()

let maplocalleader = ","

let g:vimtex_compiler_progname = 'nvr'

let g:vimtex_compiler_latexmk = {
			\ 'backend' : 'nvim',
			\ 'background' : 1,
			\ 'build_dir' : '',
			\ 'callback' : 0,
			\ 'continuous' : 0,
			\ 'executable' : 'latexmk',
			\ 'options' : [
			\   '-pdf',
			\   '-verbose',
			\   '-file-line-error',
			\   '-synctex=1',
			\   '-interaction=nonstopmode',
			\ ],
			\}
```

**Comment by lervag** [comment]
Thanks for a thorough and good issue description!

Could you check if the snippet from [this comment](https://github.com/lervag/vimtex/issues/767#issuecomment-293851817) fixes your issue?

**Comment by alefa** [comment]
Yes, that fixes it. Thanks a lot for your quick response!

**Comment by lervag** [comment]
No problem! You should note that the problem is due to a bug in neovim that should be fixed when version 0.2 is released. Then you should also remove the snippet, since it deactives a useful feature (in particular for multi-file projects, in which case paths in the quickfix list may be incorrect and the `...fix_paths` function will fix many/most of them).
